lib/model.ts

```diff
@@ -1,10 +1,12 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT license.
 
+import {flatbuffers} from 'flatbuffers';
 import {onnx} from 'onnx-proto';
 
 import {Graph} from './graph';
 import {OpSet} from './opset';
+import {onnxruntime} from './ort_format_schema';
```

Comment:
how to update ort format schema when new version is available?

Reply:
Currently there's no automated way to update it. In onnxruntime, this file is also manually updated with a command. [ORT File Format](https://github.com/microsoft/onnxruntime/tree/d42399e1b07ce61e95aae88bc6b6ea5dcaae2011/onnxruntime/core/flatbuffers/schema#ort-file-format). We'll have to keep track of the version history from onnxruntime, and update the schema accordingly. Maybe we can provide a readme with the update instructions?

Reply:
We may provide a readme and also make a note at ort document.